DQMOffline/Trigger/python/B2GMonitoring_cff.py

```diff
@@ -1,4 +1,45 @@
 import FWCore.ParameterSet.Config as cms
 
+from DQMOffline.Trigger.JetMETMonitor_cfi import hltJetMETmonitoring
+from DQMOffline.Trigger.HTMonitor_cfi import hltHTmonitoring
+
+# HLT_AK8PFHT700_TrimR0p1PT0p03Mass50                                                                                                                                         
+AK8PFHT700_HTmonitoring = hltHTmonitoring.clone()
+AK8PFHT700_HTmonitoring.FolderName = cms.string('HLT/B2GMonitor/AK8PFHT700_TrimR0p1PT0p03Mass50')
+AK8PFHT700_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_AK8PFHT700_TrimR0p1PT0p03Mass50_v*")
+AK8PFHT700_HTmonitoring.jets = cms.InputTag("ak8PFJetsCHS")
+AK8PFHT700_HTmonitoring.jetSelection      = cms.string("pt > 0 && eta < 2.5")
+AK8PFHT700_HTmonitoring.jetSelection_HT = cms.string("pt > 200 && eta < 2.5")
+
+# HLT_AK8PFJet360_TrimMass30                                                                                                                                                  
+AK8PFJet360_TrimMass30_PromptMonitoring = hltJetMETmonitoring.clone()
+AK8PFJet360_TrimMass30_PromptMonitoring.FolderName = cms.string('HLT/B2GMonitor/AK8PFJet360_TrimMass30')
+AK8PFJet360_TrimMass30_PromptMonitoring.ptcut = cms.double(360)
+AK8PFJet360_TrimMass30_Promptonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_AK8PFJet360_TrimMass30_v*")
+AK8PFJet360_TrimMass30_PromptMonitoring.pfjets      = cms.InputTag("ak8PFJetsCHS")
+AK8PFJet360_TrimMass30_PromptMonitoring.jetSelection      = cms.string("pt > 0 && eta < 2.5")
+AK8PFJet360_TrimMass30_PromptMonitoring.jetSelection_HT= cms.string("pt > 200 && eta < 2.5")
+
+
+# HLT_PFHT650_WideJetMJJ900DEtaJJ1p5                                                                                                                   
+PFHT650_WideJetMJJ900DEtaJJ1p5_HTmonitoring = hltHTmonitoring.clone()                        
+PFHT650_WideJetMJJ900DEtaJJ1p5_HTmonitoring.FolderName = cms.string('HLT/B2GMonitor/PFHT650_WideJetMJJ900DEtaJJ1p5')
+PFHT650_WideJetMJJ900DEtaJJ1p5_HTmonitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_PFHT650_WideJetMJJ900DEtaJJ1p5_v*")
+PFHT650_WideJetMJJ900DEtaJJ1p5_HTmonitoring.jets = cms.InputTag("ak8PFJetsCHS")
+PFHT650_WideJetMJJ900DEtaJJ1p5_HTmonitoring.jetSelection = cms.string("pt > 0 && eta < 2.5")
+PFHT650_WideJetMJJ900DEtaJJ1p5_HTmonitoring.jetSelection_HT =  cms.string("pt > 200 && eta < 2.5")
```

Comment:
how do you check that we do not have spurious events at low HT ?
they will not affect the analysis, but they will contribute to the rate ...